metrics/metrics_test.go

```diff
@@ -22,3 +32,94 @@ func TestMethodsDontPanic(t *testing.T) {
 	ssMetrics.AddUDPNatEntry()
 	ssMetrics.RemoveUDPNatEntry()
 }
+
+func BenchmarkGetLocation(b *testing.B) {
+	var ipCountryDB *geoip2.Reader
+	// The test data is in a git submodule that must be initialized before running the test.
```

Comment:
Please add the instructions to the README

Reply:
Done